ocaml/xapi/quicktest_raw_vdi_import.ml

```diff
@@ -0,0 +1,55 @@
+(*
```

Comment:
Just one minor comment: if we rename this file to `quicktest_import_raw_vdi` it will be easier to find based on the name of the function it tests.